rainbow-me/rainbow
# RPC Proxy

Fixes APP-####

## What changed (plus any additional context for devs)
Switches all RPC endpoints to use [RPC proxy](https://github.com/rainbow-me/arc/pull/258). Requires `rpc_proxy_enabled` firebase flag to be true. If false, falls back to preexisting RPC endpoints.

Requires:
https://github.com/rainbow-me/rainbow-scripts/pull/136
https://github.com/rainbow-me/rainbow-env/pull/51

## Screen recordings / screenshots


## What to test
1. Make sure I didn't forget to "convert" any RPC endpoints to use proxy.
2. Make sure proxy works (tx don't blow up)
